e2e/pulsar_test/pulsar_test.go

```diff
@@ -324,7 +324,7 @@ func TestSubmitCancelJobSet(t *testing.T) {
 		}
 
 		// Test that we get submit, cancel job set, and cancelled messages.
-		numEventsExpected := numJobs + 1 + numJobs
+		numEventsExpected := numJobs + numJobs + numJobs
```

Comment:
So this is a change in the api.  Previously we had two ways of cancelling a jobset via the api:
- Call CancelJob with an empty jobId
- Call CancelJobset
The two went down different code paths with the first sending a CacnelJobset message and the second sending n cancel jobs messages (one for each job in the jobset).  I've unified the codepaths so now we just do the latter.